include/mgba/core/lockstep.h

```diff
@@ -19,14 +22,20 @@ enum mLockstepPhase {
 };
 
 struct mLockstep {
+#if MGBA_LOCK_STEP_USE_MUTEX
+	Mutex mutex;
+	Condition cond;
+#endif
+
 	int attached;
-	enum mLockstepPhase transferActive;
+	int32_t transferActive; // must be one of the values in mLockstepPhase
```

Comment:
Why did you change this away from an enum?

Reply:
Win32 atomic function does not support abitrary type